backend/danswer/connectors/confluence/confluence_utils.py

```diff
@@ -0,0 +1,22 @@
+import bs4
+
+
```

Comment:
I exported this function into a utils file so when I generate the id in the permission sync and when i generate the id in the connector, they match. 

Maybe even more of the logic can be exported to here to make sure they match but this is it for now